engine/runtime-with-instruments/src/test/java/org/enso/interpreter/test/instrument/IncrementalUpdatesTest.java

```diff
@@ -227,17 +228,18 @@ private static String extractPositions(String code, String chars, Map<Character,
     return executionCompleteEvents;
   }
 
-  private List<Runtime$Api$Response> sendEditFile(String originalText, String newText) {
+  private List<Runtime$Api$Response> sendExpressionValue(String originalText, String newText) {
     assertNotNull("Main file must be defined before", mainFile);
-    context.send(Request(new Runtime$Api$EditFileNotification(
+    context.send(Request(new Runtime$Api$SetExpressionValueNotification(
```

Comment:
The test could be _duplicated_. As we can still recognize the simple `EditFileNotification` - we could have two variants of the test - one sending `SetExpressionValueNotification` and the other one `EditFileNotification`.